conformance/src/main/scala/za/co/absa/enceladus/conformance/DynamicConformanceJob.scala

```diff
@@ -97,9 +97,9 @@ object DynamicConformanceJob {
     processResult(result, performance, publishPath, stdPath)
   }
 
-  private def obtainSparkSession(cmd: CmdConfig): SparkSession = {
+  private def obtainSparkSession(): SparkSession = {
     val spark: SparkSession = SparkSession.builder()
-      .appName(s"Dynamic Conformance ${cmd.datasetName} ${cmd.datasetVersion} ${cmd.reportDate} ${cmd.reportVersion}")
+      .appName(s"Dynamic Conformance")
```

Comment:
Same question as for Standardisation. Why dataset details are removed from the app name?

Reply:
Same answer here